### PR TITLE
Pulumi README copyedits

### PR DESCRIPTION
**Pull request by sarahrivka**
### Proposed changes
Light copyedits on Pulumi ReadMe.

### Checklist
Before creating a PR, run through this checklist and mark each as complete.

- [X] I have written my commit messages in the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) format. 
- [X] I have read the [CONTRIBUTING](/CONTRIBUTING.md) doc
- [X] I have added tests (when possible) that prove my fix is effective or that my feature works
- [X] I have checked that all unit tests pass after adding my changes
- [X] I have updated necessary documentation
- [X] I have rebased my branch onto master
- [X] I will ensure my PR is targeting the master branch and pulling from my branch from my own fork

### CODE REVIEW

File: pulumi/python/README.md [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
-Pulumi supports many programming languages, Python was chosen as the language for this project. The reimplementation of
-the deployments definitions here should be reproducible in other languages.
+This project illustrates the end-to-end stand up of an AWS VPC cluster, Elastic 
+Kubernetes Service (EKS), NGINX Kubernetes Ingress Controller (KIC), and a sample 
+application using [Pulumi](https://www.pulumi.com/). It is intended to be used 
+as a reference when building your own Infrastructure as Code (IaC) deployments. 
+As such, each discrete stage of deployment is defined as a separate Pulumi project 
+that can be deployed independently of each stage. Although Pulumi supports many 
+programming languages, Python was chosen as the language for this project. 
+The reimplementation of the deployments definitions here should be reproducible
```
Comment (sarahrivka) [inline pulumi/python/README.md:10]: Is "deployments definitions" the correct phrasing? Grammatically it seems a bit off but maybe this is a common phrase.

Comment (qdzlug) [reply pulumi/python/README.md:10]: Go ahead and change this to "...reimplementation of the deployment here should be easily reproducible..."

File: pulumi/python/README.md [hunk 2]
```diff
@@ -76,105 +85,120 @@ This directory contains all the Pulumi/Python based logic, which currently consi
...
+Starting with Kubernetes version 1.22, support for the Ingress Beta API 
+`networking.k8s.io/v1beta` will be dropped, requiring use of the GA Ingress API 
+`networking.k8s.io/v1`. However, Kubernetes versions 1.19 through 1.21 allows 
+these two API versions to coexist and maintains compatibility for consumers of 
+the API – meaning, the API will respond correctly to calls to either the `v1beta` 
+and/or `v1` routes.
 
-This project uses the NGINX KIC v2.x releases which includes full support for the GA APIs. do not use the
+This project uses the NGINX KIC v2.x releases which includes full support 
+for the GA APIs. Do not use the [???]
```
[the remark this answers is not in the document]
Comment (qdzlug) [reply pulumi/python/README.md:184]: Hmmm, I wonder what I was thinking there? Just go ahead and take out the whole sentence "Do not use the...".

I think that I wanted to say that you should not use the Beta API's, but I don't know that it's necessary to draw that out.

**Review by qdzlug — CHANGES_REQUESTED** [review]
Two little notes there, but other wise looks great. I'll merge once you update with those.

Jay

File: pulumi/python/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (sarahrivka) [reply pulumi/python/README.md]: Ok great, I removed it in the new commit.

**Comment by sarahrivka** [comment]
> Two little notes there, but other wise looks great. I'll merge once you update with those.
> 
> Jay

Resolved both of these. I think I did it in the correct workflow with a new commit, but feel free to lmk if there's a better way.

**Review by qdzlug — APPROVED** [review]

**Pull request merged**